sql/core/src/test/java/test/org/apache/spark/sql/JavaDatasetSuite.java

```diff
@@ -562,79 +562,6 @@ private void assertEqualsUnorderly(
     );
   }
 
-  @Test
```

Comment:
This was actually missing piece for dropDuplicates() as well. Since we remove the functionality for batch query in dropDuplicatesWithinWatermark(), it's probably better to have another PR to deal with this.